packages/react-magma-dom/src/components/Pagination/PageButton.tsx

```diff
@@ -9,7 +9,7 @@ export interface PaginationProps extends ButtonProps {
   isSelected?: boolean;
 }
 
-function typeSize(props) {
+export function typeSize(props) {
```

Comment:
should this have a more specific name... like `paginationTypeSize` or `pageButtonTypeSize` ?